src/styles/global.css

```
@@ -2460,35 +2460,55 @@ a.tag:hover {
+  line-height: 1.04;
+  text-wrap: balance;
+}
+
+.og-heading--with-description {
+  font-size: 84px;
+  line-height: 1.04;
+  margin-bottom: 12px;
 }
 
 .og-description {
   font-family: "Inter", system-ui, sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)